index.html

```diff
@@ -1543,6 +1764,13 @@ <h3>
             The user agent MAY automatically apply the changes if the update
             does not contain changes to [=security-sensitive members=].
           </p>
+          <p>
+            If a user changes localization settings, the user agent MAY
```

Comment:
You might also want to mention that, regardless of the direction (`dir`) of the localized value, the security sensitive members should be displayed in a bidirectionally isolated way to prevent the kinds of display problems described in [Unicode Source Code Handling](https://www.unicode.org/reports/tr55) [UTR55].